pkgs/tools/security/gnupg/23.nix

```diff
@@ -0,0 +1,103 @@
+{ fetchurl, fetchpatch, lib, stdenv, pkg-config, libgcrypt, libassuan, libksba
+, libgpg-error, libiconv, npth, gettext, texinfo, buildPackages
+
+# Each of the dependencies below are optional.
+# Gnupg can be built without them at the cost of reduced functionality.
+, guiSupport ? true, enableMinimal ? false
+, adns ? null , bzip2 ? null , gnutls ? null , libusb1 ? null , openldap ? null
+, tpm2-tss ? null
```

Comment:
`tpm2-tss` is only available on Linux, so we should likely disable it on `darwin`, no?

Reply:
Please disable it with a boolean flag. Using ? null and null overrides is really error prone.